OrbitGl/TimeGraph.h

```diff
@@ -115,7 +113,6 @@ class TimeGraph {
   void SetTextRenderer(TextRenderer* text_renderer) { text_renderer_ = text_renderer; }
   [[nodiscard]] TextRenderer* GetTextRenderer() { return &text_renderer_static_; }
   void SetStringManager(std::shared_ptr<StringManager> str_manager);
-  [[nodiscard]] StringManager* GetStringManager() const;
```

Comment:
Ah, now I get why you changed it above. In this case, I'd argue that it's fine to have a `GetStringManager()` method here since this is actually a getter for a member. But I'd question if those getters are needed at all, as mentioned above.

Reply:
Agree, part of my next PR.